opus/application/apps/cart/templates/cart/cart.html

```diff
@@ -6,14 +6,22 @@ <h1>Zipped File Links</h1>
             <span class="spinner">&nbsp;</span>
         </div>
         <h1>Download Options</h1>
+        <div>
+            <button class="op-cart-select-all-btn" type="button">Select all options</button>
+            <button class="op-cart-deselect-all-btn" type="button">Deselect all options</button>
+        </div>
         <!-- <p>Total Files: <span id="total_files">{{ download_count }}</span></p> -->
         <!-- <p>Total Size (before zip): <span id="op-total-download-size">{{ total_download_size_pretty }}</span></p> -->
         <p class="op-total-size">Total Size (before zip): <span class="spinner">&nbsp;</span><span id="op-total-download-size">{{ total_download_size_pretty }}</span></p>
 
         <div id="download_options">
```

Comment:
Now would probably be a fine time to rename this to "op-download-options" since we're using it everywhere.